docs/ref/request-response.txt

```diff
@@ -735,16 +736,15 @@ Attributes
 Methods
 -------
 
-.. method:: HttpResponse.__init__(content='', content_type=None, status=200, reason=None, charset=None)
+.. method:: HttpResponse.__init__(content=b'', content_type=None, status=200, reason=None, charset=None)
 
     Instantiates an ``HttpResponse`` object with the given page content and
     content type.
 
-    ``content`` should be an iterator or a string. If it's an
-    iterator, it should return strings, and those strings will be
-    joined together to form the content of the response. If it is not
-    an iterator or a string, it will be converted to a string when
-    accessed.
+    ``content`` should be an iterator, bytestring, or a string. If it's an
```

Comment:
an iterator, bytestring, or string.
(or else "a/an" should be prefixed in all values rather than 2 of 3)